DOCS/OSI-SPDX-history.md

```diff
@@ -0,0 +1,104 @@
+
+Note that OSI has the GPL-2.0-or-later license notice on its page for the eCos exception, which would use the SPDX license expression: `GPL-2.0-or-later WITH eCos-exception-2.0`. Did OSI _only_ approve this exception for GPL-2.0-or-later? 
+
+Suggestion: Update the [eCos](https://opensource.org/licenses/eCos-2.0) page to use the current `eCos-exception-2.0` identifier and change the URL to https://opensource.org/licenses/eCos-exception-2.0. Clarify what versions of GPL it is approved for.
+
+## Licenses that allow for variations by way of a different license notice
+These are licesnes that were approved by the OSI in full. These licenses allow legally substantive variations based on use of a different license notice. This variataion warranted distinct SPDX identifiers to indicate whether or not the allowed variation is being triggered. All of these variations include a Note in the SPDX License List entry explaining this. 
+
+* `CAL-1.0-Combined-Work-Exception`
+* `MPL-2.0-no-copyleft-exception`
+* `OFL-1.1-no-RFN`
+* `OFL-1.1-RFN`
+
+Suggestion: OSI could add the variant SPDX ids to the license pages.
+
+## Not in SPDX list, but in OSI's list
+
+* [Jabber Open Source License](https://opensource.org/licenses/jabberpl) - This is a case where the text on the OSI site is not the same as the archived text at http://archive.jabber.org/core/JOSL.pdf SPDX would consider these two texts as different licenses, and was trying to clarify with the OSI which text was submitted and approved by OSI. Martin Michlmayr and Jilayne discussed this back in 2013 (https://lists.spdx.org/g/Spdx-legal/topic/22080295#475), but never reached a resolution. Jilayne emailed the OSI Board about this issue in July 2019. 
+
+Suggestion: SPDX could recognize both variants on the SPDX License List and note the one that is specifically OSI-approved is the one that matches that exact text on the OSI site?  This would require distinct names and identifiers. 
+
+* CVW: MITRE Collaborative Virtual Workspace License
+
+still need to research issue here
+
+# from Max's list - still to address: 
+## Not in OSI list, but in SPDX's list
+
+* GPL-3.0-with-GCC-exception
+
+what is the quesiton here?
+
+* LGPL-2.0
+
+this is a deprecated SPDX license id, not sure what the question is here?
+
+* MIT-Modern-Variant
+
+### False-positives
+
+- Many mismatches in GPL/LGPL/AGPL because of -only/-or-later/+. I unified those
+
+what do you mean my mismatches? and "unified those"?
```

Comment:
Yes sorry, that was unclear. Of the GPL family, I treated `-only`, `-or-later`, and `+` the same as OSI does not use these. It just means that I did not look for `GPL-2.0-or-later` on OSI's website, but just for `GPL-2.0`. Effectively I found LGPL-2.0 as missing (see the comment above).

Reply:
added note